src/main.js

```diff
@@ -41,7 +41,16 @@ uploadButton.addEventListener("click", _ => {
       )
       .then(_ => {
         uploadButton.innerText = "Uploaded";
-        setTimeout(() => (uploadButton.innerText = "Upload"), 1000);
+        setTimeout(() => {
+          uploadButton.innerText = "Upload";
+          uploadButton.disabled = false;
```

Comment:
Should this be `true`? As in  "we've uploaded the current canvas, so there's no need to upload the same thing again"

Reply:
I don't like disabling the button, as it would require the user to rebuild the card if it hadn't uploaded. Maybe we can open the grid page in a new window?